release.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -euo
+
+INTERNAL_RELEASE=true
+
+while getopts 'ph' OPTION; do
```

Comment:
Could you please describe what's happening here? As I'm not sure to be honest...

Reply:
I added comments, does it make sense now?

Reply:
Didn't see your approval :) I assume that means yes